### PR TITLE
Fixed Breadcrumb to Display Detailed Name Instead of ID

### LINKED ISSUE #9197: Fix Breadcrumb to Display Detailed Name Instead of ID
**Description:**

Currently, the breadcrumb navigation is displaying the ID of the person when navigating through pages, instead of the person's actual name. This issue can lead to confusion for users, as it is not intuitive to see an ID in the breadcrumb, which is intended to show clear and meaningful context.

And there's also a bug in the models.tsx where the patient_name was defined as number while it has to be in string.

**To Reproduce**

ISSUE 1

1. Go to 'Sample Test'
2. Click on 'Any of the Patient's Name' 
3. On the top breadcrumbs can be seen with the ID

ISSUE 2
1. Go to 'Shifting Details'
2. Click on 'All Details' 
3. On the top breadcrumbs can be seen with the ID

**Expected Behaviour:**

The breadcrumb should display the person’s full name or relevant display name (e.g., "John Doe" instead of "12345"). This will enhance user experience by making the navigation more understandable and contextually relevant.

**Screenshots**

Image

- OS: MACOS
- Browser CHROME
- Version 24

### PR DESCRIPTION
**Pull request by rayyjeb**
## Proposed Changes

- Fixes #9197 
- Change breadcrumb to display person's name instead of ID.
- Update breadcrumb component to fetch the correct name field from the data.
- Ensure breadcrumb updates dynamically based on the person’s name in relevant sections.

@ohcnetwork/care-fe-code-reviewers

## Summary by CodeRabbit

- **New Features**
	- Introduced `crumbsReplacements` for enhanced breadcrumb navigation on the Sample Details and Shift Details pages, displaying the patient's name.
	- Updated clipboard functionality in Shift Details to improve user experience.
	- Conditional rendering of the logo in printed data based on specific properties.

- **Bug Fixes**
	- Updated the `patient_name` property type in the SampleTestModel for better data handling.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -20862,4 +20862,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (Jacobjeevan) [inline package-lock.json]: Minor thing, but can you get changes in package-lock removed from the commit?

You can reset the HEAD, remove from staging and do a force push.

**Review by Jacobjeevan — CHANGES_REQUESTED** [review]

**Comment by Jacobjeevan** [comment]
@rayyjeb Ensure that other changes are in staging though.

**Comment by rayyjeb** [comment]
@Jacobjeevan Should I reopen the pull request once again since, the it is showing as closed

**Comment by Jacobjeevan** [comment]
> @Jacobjeevan Should I reopen the pull request once again since, the it is showing as closed

Yea

**Review by rithviknishad — APPROVED** [review]

**Review by Jacobjeevan — APPROVED** [review]

**Comment by rayyjeb** [comment]
@rithviknishad Could you please review the PR again? If there's anything, Do let me know

**Comment by Jacobjeevan** [comment]
> @rithviknishad Could you please review the PR again? If there's anything, Do let me know

He already approved it 🤔  and there hasn't been any changes since approval.

**Comment by rithviknishad** [comment]
@rayyjeb It's queued for testing. @nihal467 would do QA and mark it as tested. Once that's done, we are good to merge it.

**Comment by nihal467** [comment]
LGTM

**Pull request merged**